tests/data/rnx-updu_rn3214.json

```diff
@@ -0,0 +1,23 @@
+{
+    "os": {
+        "discovery": {
+            "devices": [
+                {
+                    "sysName": "<private>",
+                    "sysObjectID": ".1.3.6.1.4.1.55108.1.0",
+                    "sysDescr": "RNX UPDU RN3214 (UPDU-Right), S/N: 100124, Version 2.7.0-5741e7a3",
+                    "sysContact": "<private>",
+                    "version": null,
```

Comment:
You could extract version and serial from the sysobjectid, with the regex feature. It's in the documentation

Reply:
Added